src/tools/genpolicy/src/config_map.rs

```diff
@@ -11,10 +11,10 @@ use crate::pod;
 use crate::policy;
 use crate::settings;
 use crate::yaml;
+use crate::agent;
```

Comment:
Order alphabetically. 

If you use rust-analyzer from VSCode, that might order correctly for you.

Reply:
I do have rust-analyser and its very helpful, but mine didn't complain about the aphabetical order of imports. I might be missing a setting or another extension. Updated. Thanks!